tests/test_convert.py

```diff
@@ -94,3 +95,19 @@ def test_convert_block_commands(tmpdir, cmd):
     pyblock = convert_apdl_block(apdl_strings=apdl_block.split('\n'))
     pyblock = '\n'.join(pyblock)
     assert pymapdl_output[cmd] in pyblock
+
+
+def test_logger(caplog):
+    # I could not find a way to test the logging output. It seems that `caplog`
+    # does not work well. Hence we just run the code with `show_log` equal `True`
+    # to show it does not fail.
```

Comment:
Don't include personal comments that include "I". Instead, objectively add a comment.

Also, it would be helpful to figure out why this isn't working. If you can't debug this, ping me and I'll investigate it.

Reply:
Makes sense. I'm simplifying the comment.

Reply:
New commit uses ``capsys`` which reads the screen output (stdout). It does work.
However using the screen output might in the future break because some of the used parameters prints something. I think it is good enough for now.